src/scripting/StelMainScriptAPI.cpp

```diff
@@ -427,7 +427,10 @@ QString StelMainScriptAPI::getSkyCulture()
 void StelMainScriptAPI::setSkyCulture(const QString& id)
 {
 	GETSTELMODULE(StelObjectMgr)->unSelect(); // mistake-proofing!
-	emit requestSetSkyCulture(id);
+	if (id=="western") // for backward compatibility of scripts
```

Comment:
better use id.startsWith("western")

Reply:
I disagree, because we should run old scripts for the default SC (SCID: western) in the new environment.

Reply:
I did not check further. Does the request...(id) give an error message if western_hlad is not found? Or some other recovery? The startsWith would cause a fallback to the current modern. 
Probably also here should be a note to users: 

`qDebug() << "Western skycultures have been renamed to modern_*. Please update your scripts."`

Reply:
OK, just informing about changes and do nothing in script...